### PR TITLE
PSP saves 1st dump

### PR DESCRIPTION
**Pull request by OfficialAhmed**
A test before starting to dump more save data

### CODE REVIEW

**Comment by bucanero** [comment]
Amazing work as usual @OfficialAhmed 👍 

Just some small comments:

- the `ICON0.PNG` should be on the same level as the `saves.txt` file. For example: `/PSP/UCES01563/ICON0.PNG` . Maybe we can just move the file when running your script.
- On `saves.txt` there's a new line between the zip file and the description. It should be on the same line. For example: 
```
00000002.zip=Still in the first chapter. All levels but one are cleared.
00000099.zip=Some other description.
```

That's all, everything looks great so feel free to update the PR when you can 👍

File: PSP/LoveFinde/saves.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+00000010.zip=
+Unlocked everything
```
Comment (bucanero) [inline PSP/LoveFinde/saves.txt:2]:
```suggestion
00000010.zip=Unlocked everything
```

File: PSP/NPEZ00004/saves.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+00000004.zip=
+All levels cleared
```
Comment (bucanero) [inline PSP/NPEZ00004/saves.txt:2]:
```suggestion
00000004.zip=All levels cleared
```

**Review by OfficialAhmed — COMMENTED** [review]
All Done

**Pull request merged**